serializer/serializer.c

```diff
@@ -1,5 +1,11 @@
 #include "serializer.h"
 
+void fill_tag(char* new_tag, char* old_tag, size_t tag_len) {
+  // fill in '9'
+  memset(new_tag, 57, NUM_TRYTES_TAG);
+  memcpy(new_tag, old_tag, tag_len);
```

Comment:
Can you avoid duplicated memory operations?

Reply:
`sprintf` can deal this.
And what is the concern of duplicated memory operations? :thinking: